src/adapter.ts

```diff
@@ -47,9 +49,10 @@ export class SequelizeAdapter implements Adapter {
    * @param option sequelize connection option
    */
   public static async newAdapter(
```

Comment:
Add the new usage to README.md

https://github.com/node-casbin/sequelize-adapter#simple-example